### PR TITLE
Inline comments

### LINKED ISSUE #36: Provide inline documentation in the reactor stub
Currently the reactor stub produced via `atomkraft reactor` is completely undocumented. There should exist inline documentation in the file, explaining how to use it.

### PR DESCRIPTION
**Pull request by ivan-gavran**
When merged, this PR closes #36 .

### CODE REVIEW

**Comment by ivan-gavran** [comment]
The file generated with `atomkraft reactor --actions "Init,Transfer" --variables "var1, var2"` looks like this:

```python
'''
The reactor file connects a test scenario described by a trace
(obtained from a model or written by hand) with the actual execution
of the test scenario on the testnet.

It contains one @step function per each action appearing in the trace:
those function implement changes to the blockchain corresponding to the
abstract action from the trace.

All step functions receive the following arguments:
    testnet: a `Testnet` object on which blockchain transactions can be
             executed.
    state: additional logical state, as defined the the `state()`
           function in this file.
    action: object from the trace which corresponds to the parameters
            of the taken step.
'''

import pytest
from modelator.pytest.decorators import step


keypath = 'action'



@pytest.fixture
def state():
    '''
    Defines any additional logical state (beyond the state of the chain)
    that needs to be maintained throughout the execution. This state 
    will be passed as an argument to @step functions.
    '''
    
    #TODO: replace the empty stub object with a different state object 
    # if necessary
    return {}



@step('Init')
def init(testnet, state, var1, var2):
    '''
    This function implements the effects of the step 'Init'
    on blockchain `testnet` and state `state`. 
    It additionally has access to the model (trace) state variables `var1` and `var2`.
    '''
    
    #TODO: replace the printing stub with the effects of the action `Init`
    print("Step: Init")




@step('Transfer')
def transfer(testnet, state, var1, var2):
    '''
    This function implements the effects of the step 'Transfer'
    on blockchain `testnet` and state `state`. 
    It additionally has access to the model (trace) state variables `var1` and `var2`.
    '''
    
    #TODO: replace the printing stub with the effects of the action `Transfer`
    print("Step: Transfer")

```

There is a difference depending on how many variables are passed to `atomkraft reactor...`. If there are more than two, the function comment would enumerate variables spread into separate lines:

```python
@step('Transfer')
def transfer(testnet, state, var1, var2, var3):
    '''
    This function implements the effects of the step 'Transfer'
    on blockchain `testnet` and state `state`. 
    It additionally has access to the model (trace) state variables: 
		-`var1`
		-`var2`
		-`var3`.
    '''
    
    #TODO: replace the printing stub with the effects of the action `Transfer`
    print("Step: Transfer")

```

File: atomkraft/reactor/reactor.py [hunk 1]
```diff
@@ -77,6 +79,27 @@ def generate_reactor(
     return stub_file_path
 
 
+def _file_comment():
+    return """'''
+The reactor file connects a test scenario described by a trace
+(obtained from a model or written by hand) with the actual execution
+of the test scenario on the testnet.
+
+It contains one @step function per each action appearing in the trace:
```
Comment (rnbguy) [inline atomkraft/reactor/reactor.py]:
```suggestion
It contains one @step decorated function per each action appearing in the trace:
```

File: atomkraft/reactor/reactor.py [hunk 2]
```diff
@@ -85,12 +108,32 @@ def _keypath_stub(keypath):
...
 
+def _action_description_comment(action_name, variables):
+    if len(variables) == 0:
+        variables_sentence = ""
+    elif len(variables) == 1:
+        variables_sentence = f"It additionally has access to the model (trace) state variable `{variables[0]}`."
+    elif len(variables) == 2:
+        variables_sentence = f"It additionally has access to the model (trace) state variables `{variables[0]}` and `{variables[1]}`."
+    else:
+        vars_string = "".join(["\n\t\t-`" + v + "`" for v in variables])
```
Comment (rnbguy) [inline atomkraft/reactor/reactor.py]: Can we do?
```suggestion
        vars_string = "".join([f"\n\t\t-`{v}`" for v in variables])
```

Comment (ivan-gavran) [reply atomkraft/reactor/reactor.py]: absolutely, it's better that way

**Pull request merged**